codec/codec-sofa-hessian/src/main/java/com/alipay/sofa/rpc/codec/sofahessian/serialize/CustomThrowableGenericDeserializer.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.sofa.rpc.codec.sofahessian.serialize;
+
+import com.alipay.hessian.generic.model.GenericObject;
+
+/**
+ *
+ * @author xingqi
+ * @version : CustomThrowableGenericDeserialize.java, v 0.1 2022年10月18日 5:02 PM xingqi Exp $
+ */
+public class CustomThrowableGenericDeserializer {
+
+    private static boolean        GENERIC_THROW_EXCEPTION = "true".equalsIgnoreCase(System
+                                                              .getProperty("rpc.generic.throw.exception"));
+
+    private static final String[] THROWABLE_FIELDS        = new String[] { "cause", "detailMessage", "stackTrace",
+                                                          "suppressedExceptions" };
+
+    public static Object judgeCustomThrowable(Object appObject) {
```

Comment:
This method only identifies custom exceptions under generalized calls，so can modify appropriate method name.

Reply:
ok